README.md

```diff
@@ -64,7 +64,7 @@ Then, clone and build NovelRT:
 git clone https://github.com/NovelRT/NovelRT.git
 mkdir -p NovelRT/build
 cd NovelRT/build
-cmake ..
```

Comment:
This is not what I meant.

This is also technically wrong.

Just note we don't support GCC at this time in plain english somewhere appropriate?

Reply:
Can you clarify what you mean by this? It ran perfectly fine on my machine - also yeah, adding some explanation is probably a good idea.

Reply:
the official CXX compiler flag value is `clang++` and this is somewhat a hack on Linux. You'd usually change those env vars fully if you want to use clang.